Extended Project 3/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>{Codeable} - Werewolf</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+
+  <body>
+    <div class="limit">
+      <header>
+        <p class="name">
+          <a class="cleanLink" href="https://codeable.pe/" target="_blank"
+            ><span class="codeable1"
+              >{code<span class="codeable2">able</span>}</span
+            ></a
+          >
+        </p>
+      </header>
+      <main>
+        <h1>Grupo Werewolf</h1>
+      </main>
+      <div class="container">
+        <div class="boxed">
+          <p class="intro">
+            <span class="important"
+              >We think
+              <a class="cleanLink" href="https://codeable.pe/" target="_blank"
+                ><span class="codeable1"
+                  >{code<span class="codeable2">able</span>}</span
+                ></a
+              >
+              is: </span
+            ><br />Codeable means: A beautiful opportunity to learn new things
+            and develop new abilities. We are so excited to face new challenges
+            in this phase of our lives. We are going to do our best to absorbs
+            every piece of information that we can take. We hope to be good
+            Full-Stack Developers when we finish, and we will be ready to take
+            on new challenges.
+          </p>
+          <ul class="groupMembers">
+            <li>
+              <a href="?member=carlos" class="cleanLink">
+                <p class="memberTitle">Product owner</p>
+                <img
+                  src="https://avatars3.githubusercontent.com/u/47990946?s=460&v=4"
+                  alt="Imagen"
+                />
+                <p class="memberName">Carlos Ayala</p>
+              </a>
+            </li>
```

Comment:
The last 10 lines of code are repetitive. What do you think about use `each` method?